meson.build

```diff
@@ -0,0 +1,14 @@
+project('gtk-layer-shell', ['c'], version: '0.1.0', license: 'MIT',
+	default_options: ['c_std=c99', 'warning_level=2'])
```

Comment:
Can the flow of this and similar lines be one line per element?
```
project('gtk-layer-shell',
        ['c'],
        version: '0.1.0',
        license: 'MIT',
        default_options: ['c_std=c99', 'warning_level=2'])
```

Reply:
After we merge I'll crank warning level up to 3, add werror, and fix any problems that exposes.

Reply:
Please don't enable Werror in the file (only use it manually as a command line flag to meson).

As a package maintainer, Werror is incredibly annoying. Every time clang introduces new warnings, or code changes are made that were only tested on gcc and trip clang warnings, we need to patch things >_<

Reply:
Oh, good point. I'll turn warnings up to 3 but leave werror off.